cmake/HPX_SetupCUDA.cmake

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2019 Ste||ar-Group
+#
+# SPDX-License-Identifier: BSL-1.0
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+
+if(HPX_WITH_CUDA)
+
+  find_package(CUDA REQUIRED)
+  set(HPX_WITH_COMPUTE ON)
```

Comment:
I wonder if this one has to be `PARENT_SCOPE`... let's give it a try and fix it afterwards if needed.

Reply:
When calling `include(<cmake script>)` the scope stays identical and as `HPX_SetupCuda.cmake` is called in the top `CMakeLists.txt`, I would say there is no `PARENT_SCOPE` ^^

Reply:
Indeed, very good point. Thanks!